test/Conversion/TorchToTcp/elementwise.mlir

```diff
@@ -156,3 +156,99 @@ func.func @torch.aten.relu(%arg0: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[
   %0 = torch.aten.relu %arg0 : !torch.vtensor<[?,?],f32> -> !torch.vtensor<[?,?],f32>
   return %0 : !torch.vtensor<[?,?],f32>
 }
+
+// -----
```

Comment:
Can you add a test for lowering `ValueTensorLiteralOp`?

Reply:
This is unit-tested in the `torch.aten.batch_norm` testcase in `TorchToTcp/elementwise.mlir`. Did you want me to add a separate one anyway?

Reply:
Added it to `misc.mlir`.